src/Elcodi/Store/GeoBundle/Controller/AddressController.php

```diff
@@ -175,7 +195,8 @@ public function editAction(
     public function newAction(
         AddressInterface $address,
         FormView $formView,
-        $isValid
+        $isValid,
+        $source
```

Comment:
If the `source` route parameter can be null we should use a `$source = null` parameter here, no?